### ISSUE TITLE
Trouble with opening Behaviour Tree Viewer

### ISSUE THREAD

**Issue by jasonhbartlett** [ormesam/splatter.ai#1]
Unity version 2020.3.28
Template URP Samples

1. Created new project using the URP template.
2. Imported the package and Samples
3. Sample scene runs, no problem.
4. Attempt to view the Behaviour Tree Viewer results in error:

```csharp
ArgumentNullException: Value cannot be null.
Parameter name: styleSheet
UnityEngine.UIElements.VisualElementStyleSheetSet.Add (UnityEngine.UIElements.StyleSheet styleSheet) (at <3b9216b5eb2547d2a9ee9f63fc12b7c2>:0)
Splatter.AI.Editor.Viewer.AddStyles () (at Library/PackageCache/com.samorme.splatter.ai@07aa7664f1/Scripts/Editor/Viewer.cs:24)
Splatter.AI.Editor.Viewer.CreateGUI () (at Library/PackageCache/com.samorme.splatter.ai@07aa7664f1/Scripts/Editor/Viewer.cs:18)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <695d1cc93cca45069c528c15c9fdd749>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <695d1cc93cca45069c528c15c9fdd749>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <695d1cc93cca45069c528c15c9fdd749>:0)
UnityEditor.UIElements.DefaultEditorWindowBackend.Invoke (System.String methodName) (at <bc087a59f3c04c9aad8fcd92325a4a28>:0)
UnityEditor.UIElements.DefaultEditorWindowBackend.SendInitializeIfNecessary (System.Boolean duringOnCreate) (at <bc087a59f3c04c9aad8fcd92325a4a28>:0)
UnityEditor.UIElements.DefaultEditorWindowBackend.<SendInitializeIfNecessary>b__34_0 () (at <bc087a59f3c04c9aad8fcd92325a4a28>:0)
UnityEngine.UIElements.VisualElement+SimpleScheduledItem.PerformTimerUpdate (UnityEngine.UIElements.TimerState state) (at <3b9216b5eb2547d2a9ee9f63fc12b7c2>:0)
UnityEngine.UIElements.TimerEventScheduler.UpdateScheduledEvents () (at <3b9216b5eb2547d2a9ee9f63fc12b7c2>:0)
UnityEngine.UIElements.UIElementsUtility.UnityEngine.UIElements.IUIElementsUtility.UpdateSchedulers () (at <3b9216b5eb2547d2a9ee9f63fc12b7c2>:0)
UnityEngine.UIElements.UIEventRegistration.UpdateSchedulers () (at <3b9216b5eb2547d2a9ee9f63fc12b7c2>:0)
UnityEditor.RetainedMode.UpdateSchedulers () (at <bc087a59f3c04c9aad8fcd92325a4a28>:0)
```

Something to do with adding the style sheet.  I'll put a couple screenshots:
![image](https://user-images.githubusercontent.com/22244576/158189277-f1340bb8-dd80-4109-905d-56950764e1d3.png)
![image](https://user-images.githubusercontent.com/22244576/158189530-e08f7e2a-c3a7-4d77-a0ac-7317da3e4815.png)

![image](https://user-images.githubusercontent.com/22244576/158189368-2666f382-3db4-4da8-9d24-d8e08a425865.png)

**Comment by ormesam** [comment]
Hi Jason,

I'm on holiday at the moment but I'll have a look when I'm home. Did you change the folder structure at all?

**Comment by jasonhbartlett** [comment]
No, no folder change. No worries, appreciate the update.

**Comment by ormesam** [comment]
Hi Jason,

Tracked down the issue to the loading up of the stylesheet from a hard coded location. I have fixed the issue and created a new release [(v0.0.2)](https://github.com/ormesam/splatter.ai/releases/tag/v0.0.2) with a unitypackage for easy installation.
